.github/scripts/create-artifacts.sh

```diff
@@ -0,0 +1,31 @@
+#! /bin/bash
+rm -rf builds
+mkdir "builds"
+cd builds
+
+echo "Creating TCFV2 artifact..."
+mkdir -p tcfv2/nodejs/node_modules
+cd tcfv2
+cp ../../src/cmp_tcfv2.js nodejs/node_modules/pageLoadBlueprint.js
+zip -r nodejs.zip nodejs
+rm -rf nodejs
+cd ..
+echo "TCFV2 artifact created"
+
+echo "Creating CCPA artifact..."
+mkdir -p ccpa/nodejs/node_modules
+cd ccpa
+cp ../../src/cmp_ccpa.js nodejs/node_modules/pageLoadBlueprint.js
+zip -r nodejs.zip nodejs
+rm -rf nodejs
+cd ..
+echo "CCPA artifact created"
+
+echo "Creating AUS artifact..."
+mkdir -p aus/nodejs/node_modules
+cd aus
+cp ../../src/cmp_aus.js nodejs/node_modules/pageLoadBlueprint.js
+zip -r nodejs.zip nodejs
+rm -rf nodejs
+cd ..
```

Comment:
Could the artifact creation be extracted to a function which gets passed 'aus', 'ccpa' or 'tcfv2' ?

Reply:
Absolutely. Unfortunately my bash skills aren't up to par so couldn't get it working. Feel free to make this change yourself if you're handy with bash scripts.

Reply:
Sure np lets create a card to do this as a follow up 👍